Sources/FoundationEssentials/JSON/OutputBuffer.swift

```diff
@@ -0,0 +1,250 @@
+            initialized &+= 1
+        }
+    }
+
+    mutating func append(
+        fromContentsOf source: some Collection<T>
+    ) {
+        let count = source.withContiguousStorageIfAvailable {
+            guard let sourceAddress = $0.baseAddress, !$0.isEmpty else {
+                return 0
+            }
+            let available = capacity &- initialized
+            precondition(
+                $0.count <= available,
+                "buffer cannot contain every element from source."
+            )
+            let tail = start.advanced(by: initialized)
+            tail.initialize(from: sourceAddress, count: $0.count)
+            return $0.count
+        }
+        if let count {
+            initialized &+= count
+            return
+        }
+
+        let available = capacity &- initialized
+        let tail = start.advanced(by: initialized)
+        let suffix = UnsafeMutableBufferPointer(start: tail, count: available)
+        var (iterator, copied) = source._copyContents(initializing: suffix)
+        precondition(
+            iterator.next() == nil,
+            "buffer cannot contain every element from source."
+        )
+        assert(initialized + copied <= capacity)
+        initialized &+= copied
+    }
+
+    mutating func moveAppend(
+        fromContentsOf source: UnsafeMutableBufferPointer<T>
+    ) {
+        guard let sourceAddress = source.baseAddress, !source.isEmpty else {
+            return
+        }
+        let available = capacity &- initialized
+        precondition(
+            source.count <= available,
+            "buffer cannot contain every element from source."
+        )
+        let tail = start.advanced(by: initialized)
+        tail.moveInitialize(from: sourceAddress, count: source.count)
+        initialized &+= source.count
+    }
+
+    mutating func moveAppend(
+        fromContentsOf source: Slice<UnsafeMutableBufferPointer<T>>
+    ) {
+        moveAppend(fromContentsOf: UnsafeMutableBufferPointer(rebasing: source))
+    }
+}
+
+extension OutputBuffer<UInt8> /* where T: BitwiseCopyable */ {
+
+    mutating func appendBytes<Value /*: BitwiseCopyable */>(
+        of value: borrowing Value, as: Value.Type
+    ) {
+        let (q,r) = MemoryLayout<Value>.stride.quotientAndRemainder(
+            dividingBy: MemoryLayout<T>.stride
+        )
+        precondition(
+            r == 0, "Stride of Value must be divisible by stride of Element"
+        )
+        precondition(
+            (capacity &- initialized) >= q,
+            "buffer cannot contain every byte of value."
+        )
+        let p = UnsafeMutableRawPointer(start.advanced(by: initialized))
+        p.storeBytes(of: value, as: Value.self)
+        initialized &+= q
+    }
+}
+
+extension OutputBuffer {
+    var initializedPrefix: /*borrowed*/ BufferView<T> {
+        /* _read */ get /* borrowing(self) */ {
+            /* yield */ return BufferView(
```

Comment:
Delete commentaries ?
```suggestion
    var initializedPrefix: /*borrowed*/ BufferView<T> {
        /* _read */ get /* borrowing(self) */ {
            /* yield */ return BufferView(
```